docs/stdlib/json.rst

```diff
@@ -219,15 +216,13 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
 .. eql:operator:: jsonplus: json ++ json -> json
 
-    JSON concatenation.
-
-    JSON arrays, objects and strings can be concatenated with JSON values of
-    the same type into a new JSON value.
-
-    If you concatenate two JSON objects, you get a new object whose keys will
-    be a union of the keys of the input objects. If a key is present in both
-    objects, the value from the second object is taken.
+    Concatenates given types of :eql:type:`json` and :eql:type:`Object`
+    into one.
```

Comment:
```suggestion
    Concatenates two :eql:type:`json` arrays, objects, or string values into one.
```

This operator doesn't pertain to our `Object` type but rather to objects represented in a `json` value.